### PR TITLE
Nah setup holiday road

### PR DESCRIPTION
**Pull request by nathanRomad**
# Description

I added modules for AttractionProvider.js, AttractionSelect.js Attraction.js.
There is some basic starter code for me to reference. 
I'm going to open a new branch to work on the functionality of the attraction/"Bizarreries" dropdown.

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

-----------------------------------
# Testing Instructions
git fetch --all
git checkout nah-setupHolidayRoad
serve
-----------------------------------

The functionality is not complete and testing for functionality is not ready.  Please review the basic starter code and make sure I'm on the right track.. 

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings or errors
- [N/A] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by yasmeencole — DISMISSED** [review]
looks good

**Review by kylieanyce — DISMISSED** [review]

File: scripts/.Settings.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const settings = {
-	graphhopperKey: "",
-	npsKey: "",
-	weatherKey: ""
+	graphhopperKey: "b9444e6b-2fec-424b-8c2e-4ae00f87c3f3",
+	npsKey: "zflqCsoWMQi4mVagiujDzfWb1TZZxTAAbkviJ1Hk",
```
Comment (calebsjames) [inline scripts/.Settings.js:3]: The NPS API key should be iS64Pl03HwTtlj5MKPdCon0gBhpKACS3fsxtq0kv

**Review by calebsjames — CHANGES_REQUESTED** [review]
The API key for NPS had to be renewed. The new one is iS64Pl03HwTtlj5MKPdCon0gBhpKACS3fsxtq0kv

File: scripts/.Settings.js (hunk 1 above)
Comment (nathanRomad) [reply scripts/.Settings.js:3]: changes made. I will re-submit my request.

**Review by nathanRomad — COMMENTED** [review]
all changes made, conflicts resolved.  

I will push my branch up again and create a new pull request.

**Review by calebsjames — APPROVED** [review]
Looks great!

**Review by yasmeencole — APPROVED** [review]
looks good

**Pull request merged**